api/v1beta2/ibmpowervscluster_types.go

```diff
@@ -251,6 +251,9 @@ type TransitGateway struct {
 	// id of resource.
 	// +optional
 	ID *string `json:"id,omitempty"`
+	// globalRouting indicates whether to set global routing true or not while creating the transit gateway.
```

Comment:
Lets also mention a liner about global routing and local routing and when to use which. 
Also what system will do when the field is omitted like as we mentioned in other api types. Lets keep as descriptive as possible.

Reply:
Done